liquibase.yaml

```
@@ -0,0 +1,176 @@
+
+  - uses: strip
+
+subpackages:
+  - name: "${{package.name}}-compat"
+    description: "Compat package for Liquibase Symlinks"
+    pipeline:
+      - runs: |
+          # Symlinks for liquibase
+          mkdir -p ${{targets.contextdir}}/usr/local/bin
+          ln -sf /usr/share/java/liquibase ${{targets.contextdir}}/liquibase
+          ln -sf /usr/share/java/liquibase/liquibase ${{targets.contextdir}}/usr/local/bin/liquibase
+          ln -sf /usr/share/java/liquibase/docker-entrypoint.sh ${{targets.contextdir}}/usr/local/bin/docker-entrypoint.sh
+
+update:
+  enabled: true
+  github:
+    identifier: liquibase/liquibase
+    strip-prefix: v
+
+test:
+  environment:
+    environment:
+      SQLITE_URL: jdbc:sqlite:/tmp/test.db
+      LIQUIBASE_HOME: /usr/share/java/liquibase
+    contents:
+      packages:
+        - sqlite
+        - liquibase-package-manager
+        - ${{package.name}}-compat
+  pipeline:
+    - name: Test CLI basics
+      runs: |
+        liquibase --version
+    - name: Test Liquibase Changelogs
+      runs: |
+        # Setup sqlite database
+        sqlite3 /tmp/test.db "CREATE TABLE test (id INT, name TEXT); INSERT INTO test VALUES (1, 'hello');"
+
+        # Setup liquibase test project dir
+        mkdir -p $LIQUIBASE_HOME/test
+
+        # Create changlog file
+        cat <<'EOF' >> $LIQUIBASE_HOME/test/example-changelog.sql
+        --liquibase formatted sql
+
+        --changeset your.name:1 labels:example-label context:example-context
+        --comment: example comment
+        create table person (
+            id integer primary key autoincrement not null,
+            name varchar(50) not null,
+            address1 varchar(50),
+            address2 varchar(50),
+            city varchar(30)
+        )
+        --rollback DROP TABLE person;
+
+        --changeset your.name:2 labels:example-label context:example-context
+        --comment: example comment
+        create table company (
+            id integer primary key autoincrement not null,
+            name varchar(50) not null,
+            address1 varchar(50),
+            address2 varchar(50),
+            city varchar(30)
+        )
+        --rollback DROP TABLE company;
+
+        --changeset other.dev:3 labels:example-label context:example-context
+        --comment: example comment
+        alter table person add column country varchar(2)
+        --rollback ALTER TABLE person DROP COLUMN country;
+        EOF
+
+        cd $LIQUIBASE_HOME
+
+        # Check status
+        liquibase status --url="$SQLITE_URL" --changelog-file=test/example-changelog.sql
+
+        # Inspect sql
+        liquibase update-sql --url="$SQLITE_URL" --changelog-file=test/example-changelog.sql
+
+        # Run changlog update
+        liquibase update --url="$SQLITE_URL" --changelog-file=test/example-changelog.sql
+
+        # Verify tables were created
+        sqlite3 /tmp/test.db ".tables" | grep -E "company|person"
```

```suggestion
        sqlite3 /tmp/test.db ".tables" | grep -q "company"
        sqlite3 /tmp/test.db ".tables" | grep -q "person"
```

This way it will be AND-ed